src/content/en/2020/seo.md

```diff
@@ -23,3 +23,835 @@ queries: 07_SEO
+
+{{ figure_markup(
+  image="seo-canonical-implementation-method.png",
+  caption="Canonical Implementation Method",
+  description="Bar chart showing the canonical tag implementation method. We observed that only 1.03% of desktop homepages and 0.88% of mobile ones are relying on the HTTP headers for their implementation, meaning that canonical tags are prominently implemented via the HTML head of a page.",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vTeddX0_5hUvxpYkd-927hEBlIqzuQiFn5450F2gNc9RJ5K8Wy0ln3KKD-gPWAxJ5Lo3H_km4ljHQEt/pubchart?oid=542127514&format=interactive",
+  sheets_gid="337739550",
+  sql_file="pages_wpt_bodies_by_device.sqll"
+  )
+}}
+
+When analyzing the canonical tag implemented in the raw HTML vs. those relying on client side JavaScript rendering, it was identified that 0.68% of the mobile home pages and 0.54% of the desktop ones are detected to be featuring a canonical tag in the rendered but not the raw HTML. This means that there’s only a very small number of pages that are relying on JavaScript to implement canonical tags.
+
+On the other hand, in 0.93% of the mobile home pages and 0.76% of the desktop ones it was seen that there are canonical tags implemented via both the raw and the rendered HTML and there’s a conflict between the URL specified in the raw vs. the rendered HTML of the same pages, which can generate issues.
+
+A similar conflict can be found with the different implementation methods, with 0.15% of the mobile home pages and 0.17% of the desktop ones showing conflicts between the canonical tags implemented via their HTTP headers and HTML head.
+
+### Content
+
+The primary purpose that both search engines and Search Engine Optimization serve is to give visibility to content that users need. Search engines extract features from pages to determine what the content is about. In that way the two are symbiotic. The features extracted align with signals that indicate relevance and inform ranking.
+
+To understand what search engines are able to effectively extract, we have broken out the components of that content and examined the incidence rate of those features between the mobile and desktop contexts. We also reviewed the disparity between mobile and desktop content. The mobile vs desktop disparity is especially valuable because Google has moved to Mobile First Indexing (MFI) for all new sites and as of March of 2021 will move to a mobile-only index wherein content that does not appear within the mobile context will not be evaluated for ranking.
+
+#### Text
```

Comment:
Btw @aleyda i wasn’t sure what this was about but this was in the Google Doc. Should it be removed?

Also can you check the Table of Contents to make sure all headings have the appropriate level? Some looked a bit weird to me (should Links be under meta data?) but I copied them as they where in the Doc. Including this one.

Reply:
Hi @bazzadp - could you please clarify what you're referring that if it has to be left or removed? Do you mean the intro to the content section? 

I'll take a look at the headings! Thanks for the heads up :)

Reply:
The heading Text:

![image](https://user-images.githubusercontent.com/10931297/100545238-c2181f80-3252-11eb-80b4-24231d3fa684.png)

This also appears in the Google Doc so I copied it over as was but presume either some content for this heading is missing or this heading shouldn't be here?:

![image](https://user-images.githubusercontent.com/10931297/100545256-da883a00-3252-11eb-9e92-39d6b2bb640c.png)